sdk/js/src/service/devices/index.js

```diff
@@ -275,74 +274,11 @@ class Devices {
     const dev = device
 
     if (abp) {
-      const session = {
```

Comment:
Removing this chunk will make the SDK behavior deviate from the one of the [CLI implementation](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/cmd/ttn-lw-cli/commands/end_devices.go#L332-L405).
The discussion in https://github.com/TheThingsNetwork/lorawan-stack/issues/654 did not make it entirely clear to me whether we want to keep this functionality in the SDK or not.

@htdvisser, can you have your say?

I just want to make sure we're doing the correct thing with removing this.

Reply:
The whole idea of this PR is to move generation to the client, in this case, the console.
The only difference I see that the console is not setting `root_key_id`. 
@htdvisser Should we append it to the payload? Something like `ttn-lw-console-generated`should suffice.

Reply:
If you see the SDK as a standalone entity (and not only in conjunction with the console), I think it can make sense to consider matching its behavior with the CLI. 
I just want to be clear about whether we should think this way or not; both is fine with me.

Reply:
The CLI is not an SDK, it's a client, just like the Console.

The purpose of any SDK is to provide a (ideally thin) language-specific interface on top of our HTTP and/or gRPC API. Some common business logic (like the decision to talk to the JS or not when creating an OTAA/ABP device) can be handled by SDKs. SDKs may also provide helpers for the client (such as generating random keys), but that indeed doesn't belong here.

Reply:
@htdvisser What about
> Should we append it to the payload? Something like ttn-lw-console-generatedshould suffice.